app/views/admin/invoices/show.html.erb

```diff
@@ -0,0 +1,36 @@
+<h1> admin invoice show </h1>
+
+<section id="invoice-information">
+  <ul>
+    <li>invoice id: <%= @invoice.id %></li>
+    <li>invoice status: <%= @invoice.status %></li>
+    <li>invoice creation date: <%= @invoice.created_at.strftime("%A, %B %d, %Y") %></li>
+    <li>invoice total: <%= @invoice.invoice_items.invoice_amount %></li>
+  </ul>
+
+  <%= form_with invoice: @invoice, method: :patch do |f| %>
+    <%= f.select :status, Invoice.statuses.keys, selected: "#{@invoice.status}" %>
+    <%= f.label :status %>
+    <%= f.submit %>
+  <% end %>
```

Comment:
This is the partial Max was talking about; I'm assuming he will want to refactor this section to use the partial